src/app/DTT/select/index.js

```diff
@@ -18,6 +18,7 @@ const Write = ({
 }) => {
   const options = useSelector(selectCode(groupCode)) || []
 
+  const defaultValue = safelyParseJson(data?.value).toString()
```

Comment:
What is this meant to do? Why are we calling .toString() on this?

Reply:
Because the select component doesn't take array as the default value when the value of multiple is not set to true. ie when multiple={true} props is not passed in the select. It was causing an error.  We were getting those single value stored in an array. Therefore, I changed it to string.